### ISSUE TITLE
Firmware update problem: GLIBC_2.28 not found

### ISSUE THREAD

**Issue by crogre** [marthoc/docker-deconz#260]
When I try to follow the update instructions on the home page, I get dynamic linkage problems:

```
# /usr/bin/GCFFlasher_internal

/usr/bin/GCFFlasher_internal.bin: /lib/arm-linux-gnueabihf/libc.so.6: version `GLIBC_2.28' not found 
(required by /usr/lib/libwiringPi.so)
```
It seems `libwiringPi.so` is linked with 2.28 but 2.24 is in the docker image.

**Comment by manup** [comment]
Which GCFFlasher version is this? With GCFFlasher 3.15 libwiringPi is optional and only checked at runtime.

**Comment by crogre** [comment]
How do I query the version? This is on the latest docker, image f0d5b44e5960, (app version 2.05.82 / 9/14/2020)

**Comment by githtz** [comment]
I guess this is a duplicate of https://github.com/marthoc/docker-deconz/issues/236

**Comment by manup** [comment]
Can you please try to rebuild the container locally:

```
docker stop deconz

git clone https://github.com/marthoc/docker-deconz.git
cd docker-deconz
docker build -t "user/deconz" ./armhf

```

And then start this container `user/deconz`.

I think the image on Docker Hub, still contains GCFFlasher 3.14 and 3.15 was activated in the second build (the first build was a bit messy :/

**Comment by crogre** [comment]
That works! Maybe it would be good to pin the version of WiringPi in the docker (instead of downloading latest)?

**Comment by manup** [comment]
Perfect good to know, the wiringPi version is ok to be latest, it has a very stable api. The problem was that GCFFlasher versions before 3.15 linked to a specify version which is fixed in 3.15.

**Comment by marthoc** [comment]
Addressed in #265.
